### DISCUSSION TITLE
taiga-ui vs. material

### DISCUSSION THREAD

**Discussion by manoharnikkam** [taiga-family/taiga-ui#75] [Q&A] [answered]
How is taiga-ui components different from [material design](https://material.io/components)? What are benefits?

**Comment by waterplea** [comment]
Hello there.

These are pretty different libraries. Taiga has more components and somewhat different approach to customization through ng-polymorpheus. I haven't really took the time to create any comparison because I'm sure it's worth it. But if you are interested in particular subjects I can go over them if you like. We have a little show and tell + Q&A session on YouTube, maybe you can start from there to see if Taiga UI is what you need:
https://www.youtube.com/watch?v=KdO8CFCXQu0

**Comment by WynieCronje** [comment, marked as answer]
I found a nice answer on [#taiga-ui](https://discord.gg/zrB2EdJjEy) discord channel:
![image](https://user-images.githubusercontent.com/4537265/105966600-8ef4df80-608d-11eb-8249-168d4ae601a7.png)
